eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -387,12 +393,30 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
-  $pkgList = [array]@()
+# Retrieve release tag for artiface package. If multiple packages, then output the first one.
+function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+  if (!$artifactLocation) {
+    return ""
+  }
+  try {
+    $pkgs, $ParsePkgInfoFn = RetrivePackages -pkgRepository $pkgRepository -artifactLocation $artifactLocation
+    if (!$pkgs -or !$pkgs[0]) {
+      return ""
+    }
+    $parsedPackage = &$ParsePkgInfoFn -pkg $pkgs[0]
+    return $parsedPackage.ReleaseTag
+  }
+  catch {
+    if ($continueOnError) {
+      return ""
+    }
+    Write-Error "No release tag retrieved from $artifactLocation"
+  }
+}
+function RetrivePackages($pkgRepository, $artifactLocation) {
   $ParsePkgInfoFn = ""
```

Comment:
Spelling

Reply:
Which spelling is wrong?

Reply:
I have changed from capital to lower case

Reply:
RetrivePackages -> RetrievePackages

Reply:
Thanks! Will make the change